src/@crema/components/AppPage/index.tsx

```
@@ -0,0 +1,261 @@
+    const [addEditForm] = Form.useForm();
+    const { messages: t } = useIntl();
+
+    const [visibleModal, setVisibleModal] = useState<boolean>(false);
+    const [selectedItem, setSelectedItem] = useState<any | null>(null);
+    const isEdit = useMemo(() => selectedItem !== null, [selectedItem]);
+
+    // ---------- Base API (TanStack) ----------
+    const baseApi = useBaseApiHooks<any, any, any>(endpoint || '');
+    // Params dùng cho useGetItems, thay đổi sẽ trigger query refetch
+    const [listParams, setListParams] = useState<Record<string, any>>(defaultParams ?? {});
+    const listQuery = endpoint ? baseApi.useGetItems(listParams) : api?.useGetItems?.(listParams);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)